src/v2/cookbook/cookbook-contributions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Cookbook Contributions
+type: cookbook
+order: 1000
+---
+
+## What we're looking for
+
+The Cookbook gives developers examples to work off of that both cover common or interesting use cases, and also progressively explain more complex detail. Our goal is to move beyond a simple introductory example, and demonstrate concepts that are more widely applicable, as well as some caveats to the approach. If you're interested in contributing, please follow the template below as much as possible. Some sections are required, and some are optional.
+
+## Simple Example
+
+_required_
+
+- Articulate the problem in a sentence or two.
+- Explain the simplest possible solution in a sentence or two.
+- Show a small code sample.
+- Explain what this accomplishes in a sentence.
```

Comment:
Hm, sometimes I like to include a code example demonstrating the problem before I go into the solution.

Also, maybe these should be numbered to imply that they are all required and should generally be followed in order.

Reply:
Hm, what if the flow of how to explain that code in particular doesn't work that way? Perhaps it's possible to do ol and say that the order is "strongly suggested, but not imperative". Actually imperative has other meanings in code haha, so maybe... "vital".